src/main/kotlin/no/nav/familie/ks/sak/kjerne/endretutbetaling/EndretUtbetalingAndelService.kt

```diff
@@ -138,6 +144,40 @@ class EndretUtbetalingAndelService(
             it.copy(id = 0, behandlingId = behandling.id, erEksplisittAvslagPåSøknad = false)
         endretUtbetalingAndelRepository.save(kopiertOverEndretUtbetalingAndel)
     }
+
+    fun sanityBegrunnelseTilRestFormat(
```

Comment:
```suggestion
  private  fun sanityBegrunnelseTilRestFormat(
```

Kan gjøres om til private siden den bare brukes internt